### DISCUSSION TITLE
TODO：添加水印，维护成果

### DISCUSSION THREAD

**Discussion by didilinkin** [WeNeedHome/SummaryOfLoanSuspension#911] [Ideas]
现发现有微信公众号在使用本项目生成的地图图片做付费阅读

本来这个项目应该是免费 群策群力 互帮互助的， 但是有人用这个作为创收工具，内心无法接受，这不应该是一个付费项目

![image](https://user-images.githubusercontent.com/16011235/179727701-94411b84-79cd-4eb4-bba9-d7b56bc14cf6.png)

![image](https://user-images.githubusercontent.com/16011235/179727732-235b8d4b-5d55-45de-a13d-feb47bde6fce.png)

**Comment by ghost** [comment]
支那人就是这样，任何一点东西都必须标个价，生怕自己损失一点什么。

**Comment by MarkShawn2020** [comment]
可以，不能让我们的心血被糟蹋

**Comment by MarkShawn2020** [comment]
已搞定了水印
